src/domain/cpu/index.ts

```diff
@@ -0,0 +1,25 @@
+import Worker from "worker-loader!./workers/hello.worker";
+import { DataInMessagePayload } from "./workers/hello.worker";
```

Comment:
We should also use DataOutMessagePayload

Reply:
We do use it, just not in this "sandbox" testing code. It is defined and ready to use in the future implementation.